modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -49,4 +49,60 @@
       end
     end
   end
+
+  factory :auto_established_claim_without_flashes, class: 'ClaimsApi::AutoEstablishedClaim' do
+    id { SecureRandom.uuid }
+    status { 'pending' }
+    source { 'oddball' }
+    evss_id { nil }
+    auth_headers { { test: ('a'..'z').to_a.shuffle.join } }
+    form_data do
+      json = JSON.parse(File.read("#{::Rails.root}/modules/claims_api/spec/fixtures/form_526_no_flashes_json_api.json"))
+      json['data']['attributes']
+    end
+    flashes { form_data.dig('veteran', 'flashes') }
+    special_issues do
+      if form_data['disabilities'].present? && form_data['disabilities'].first['specialIssues'].present?
+        mapper = ClaimsApi::SpecialIssueMappers::Bgs.new
+        [{ code: form_data['disabilities'].first['diagnosticCode'],
+           name: form_data['disabilities'].first['name'],
+           special_issues: form_data['disabilities'].first['specialIssues'].map { |si| mapper.code_from_name!(si) } }]
+      else
+        []
+      end
+    end
+
+    trait :status_errored do
+      status { 'errored' }
+      evss_response { 'something' }
+    end
+  end
+
+  factory :auto_established_claim_without_special_issues, class: 'ClaimsApi::AutoEstablishedClaim' do
```

Comment:
Totally possible I'm missing something, but is this factory the exact same as the `auto_established_claim_without_flashes` factory?